desk.go

```diff
@@ -201,6 +201,22 @@ func (l *deskLayout) scaleVars(scale float32) []string {
 	}
 }
 
+// MouseInNotify can be called by the window manager to alert the desktop that the cursor has entered the canvas
+func (l *deskLayout) MouseInNotify(x int, y int) {
+	barX, barY := appBar.Position().X, appBar.Position().Y
+	barWidth, barHeight := appBar.Size().Width, appBar.Size().Height
+	if x >= barX && x <= barX+barWidth {
```

Comment:
I have verified that the canvas scale != 1 does in fact break this (it is comparing screen coordinates with fyne canvas positions).
My suggestion would be that as this API is outside of the wm package it should take a fyne.Position. Then the callers of this would have to scale appropriately.